### DISCUSSION TITLE
Team is added by default when group enabled site is created

### DISCUSSION THREAD

**Discussion by web265p3** [pnp/pnpcore#1245] [General]
I noticed today that it seems to be not possible anymore to just create a group without a team.

The following code lines create a site with a team connected to it by default:
```

// Create group connected team site, using application permissions
var teamSiteToCreate = new TeamSiteOptions("mynewsite", "My new site")
{
    Description = "My site description",
    IsPublic = true,
    Owners = new string[] { "ann@contoso.onmicrosoft.com" }
};

SiteCreationOptions siteCreationOptions = new SiteCreationOptions()
{
    UsingApplicationPermissions = true
};

using (var newSiteContext = await context.GetSiteCollectionManager().CreateSiteCollectionAsync(teamSiteToCreate, siteCreationOptions))
{
    // Do work on the created site collection via the newSiteContext
};
```

The documentation [states still that only a site gets ](https://pnp.github.io/pnpcore/using-the-sdk/admin-sharepoint-sites.html)created. 

Has this changed? It makes not much sense to create group sites without teams I guess... but I wanted to clarify my understanding.

A while ago I had to "teamify" each group after creation...

**Comment by web265p3** [comment]
Now - 1 hour later - I executed exaclty the same code with the same version under the same conditions and no team is attached. It seems that MS are changing the behaviour of their API somehow... can this be?

**Reply by jansenbe** [reply]
@web265p3 : might have been some hickup or change that got pulled back. Everything still working as expected at the moment?

**Reply by web265p3** [reply]
I experienced the same again on the next day - sometimes a team gets created, but most of the times not. I now check after each call what happened and then I create a team myself or keep it like it is. I will report here if this stabilizes.
